testing/virt-what/APKBUILD

```diff
@@ -2,7 +2,7 @@
 # Maintainer:
 pkgname=virt-what
 pkgver=1.15
-pkgrel=0
+pkgrel=1
```

Comment:
There don't seem to be functional changes in the resulting package here, so there is no need to bump `$pkgrel`.